templates/cluster-template-rke2.yaml

```diff
@@ -217,7 +217,7 @@ data:
               env:
                 - name: ADDRESS
                   value: /csi/csi.sock
-              image: longhornio/csi-node-driver-registrar:v1.2.0-lh1
+              image: longhornio/csi-node-driver-registrar:v2.9.2
```

Comment:
Again, thank you for your contributions, it is always welcome and appreciated!

So, you are basically right on the need to upgrade this stuff, but I checked out the last version of the Harvester CSI provider, which is `v0.1.17` at this point in time and the image versions are different:
```yaml
csi:
    nodeDriverRegistrar:
      repository: rancher/mirrored-longhornio-csi-node-driver-registrar
      tag: v2.3.0
    resizer:
      repository: rancher/mirrored-longhornio-csi-resizer
      tag: v1.2.0
    provisioner:
      repository: rancher/mirrored-longhornio-csi-provisioner
      tag: v2.1.2
    attacher:
      repository: rancher/mirrored-longhornio-csi-attacher
      tag: v3.2.1
```

So, I will prefer these one. And, I will need to do some testing as well.